<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Show</class>
 <widget class="QDialog" name="Show">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">    background-color: #5AB2FF;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="comboBox_2">
     <property name="styleSheet">
      <string notr="true">border: 2px solid #1E0342;
background-color: #3572EF;
color: #1E0342;
font:30px;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: #1E0342;
font-weight: bold;
font-size: 30px;
background: transparent; 
border: none;
</string>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_Pass">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;
font-size: 45px;
border-radius: 50px; 
border: 2px solid #1E0342;
background-color: #A0DEFF;
color: #1E0342;
</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color: #1E0342;
font-weight: bold;
font-size: 30px;
background: transparent; 
border: none;
</string>
       </property>
       <property name="text">
        <string>National number</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_NN">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;
font-size: 45px;
border-radius: 50px; 
border: 2px solid #1E0342;
background-color: #A0DEFF;
color: #1E0342;
</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: #1E0342;
font-weight: bold;
font-size: 30px;
background: transparent; 
border: none;
</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit_Name">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;
font-size: 45px;
border-radius: 50px; 
border: 2px solid #1E0342;
background-color: #A0DEFF;
color: #1E0342;
</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="checkBox">
       <property name="styleSheet">
        <string notr="true">color: #1E0342;
font-weight: bold;
font-size: 20px;
border: none;
</string>
       </property>
       <property name="text">
        <string>Show password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="styleSheet">
      <string notr="true">border: 2px solid #1E0342;
background-color: #3572EF;
color: #1E0342;
font:30px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">border: 2px solid #1E0342;
background-color: #A0DEFF;
color: #1E0342;
font:30px;</string>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="styleSheet">
      <string notr="true">border: 2px solid #1E0342;
background-color: #A0DEFF;
color: #1E0342;
font:30px;</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
